src/screens/TokenSwap.js

```diff
@@ -190,6 +205,21 @@ const TokenSwap = () => {
     dispatch(tokenSwapSwitchTokens());
   };
 
+  const checkQuotedAmount = () => {
```

Comment:
suggestion: Add docstrings to this method, explaining what this return boolean means.